cylc/uiserver/resolvers.py

```diff
@@ -224,29 +227,3 @@ async def service(self, info, *m_args):
             self.workflows_mgr,
             log=self.log
         )
-
-    async def nodes_mutator(self, info, *m_args):
```

Comment:
@dwsutherland, any chance you could double check that this is unused code. I couldn't get any breakpoints stopping here, can't find any code calls. Both myself and @oliver-sanders think it is not used and should be removed? Also, removed its cylc-flow partner in https://github.com/cylc/cylc-flow/pull/4522/files.

Reply:
It was in use, until:
https://github.com/cylc/cylc-flow/pull/3469
specifically:
https://github.com/cylc/cylc-flow/pull/3469/files#diff-97f4d62be41218e2d70a203edfca6d44d7c0046b6908cefa8e3af7c02cd31920

And the idea behind it was that mutations being multi-workflow could be given node ids from different workflows, and then these ids would be parsed and mutations sent to the appropriate workflow..

The fact that we moved away from this, means we are happy with not using this functionality... i.e. a node mutation will always be associated with only one workflow and/or the workflow parsing happens somewhere else.